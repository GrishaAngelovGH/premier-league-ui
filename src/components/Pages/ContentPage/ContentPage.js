import PageHeader from './PageHeader'

const ContentPage = () => (
    <div className='row no-gutters'>
        <div className='col-md-12'>
            <PageHeader />
        </div>
    </div>
)

export default ContentPage