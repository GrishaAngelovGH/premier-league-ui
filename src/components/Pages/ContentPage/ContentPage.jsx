import PageHeader from './PageHeader'
import PageBody from './PageBody'

const ContentPage = () => (
    <div className='row no-gutters'>
        <div className='col-md-12'>
            <PageHeader />
            <PageBody />
        </div>
    </div>
)

export default ContentPage