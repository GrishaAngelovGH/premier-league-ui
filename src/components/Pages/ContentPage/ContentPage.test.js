import { render } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'

import ContentPage from './ContentPage'

test('should render component', () => {
    const container = render(
        <MemoryRouter initialEntries={['/matches']}>
            <ContentPage />
        </MemoryRouter>
    )

    expect(container).toMatchSnapshot()
})