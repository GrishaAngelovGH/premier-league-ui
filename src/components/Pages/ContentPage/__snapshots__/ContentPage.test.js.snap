// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="row no-gutters"
      >
        <div
          class="col-md-12"
        >
          <div
            class="row flex-nowrap overflow-auto no-gutters content-header"
          >
            <div
              class="d-flex"
            >
              <a
                href="/"
              >
                <svg
                  class="bi bi-arrow-left-short"
                  fill="white"
                  height="2em"
                  viewBox="0 0 16 16"
                  width="2em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M12 8a.5.5 0 0 1-.5.5H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5a.5.5 0 0 1 .5.5z"
                    fill-rule="evenodd"
                  />
                </svg>
              </a>
              <div
                class="m-1"
              >
                Premier League
              </div>
            </div>
            <div
              class="row flex-nowrap overflow-auto"
            >
              <div
                class="col-md-12 mt-2"
              >
                <div
                  class="row flex-nowrap overflow-auto ml-1"
                >
                  <a
                    href="/content/matches"
                  >
                    <div
                      class="border-bottom border-white item"
                      id="0"
                    >
                      matches
                    </div>
                  </a>
                  <a
                    href="/content/news"
                  >
                    <div
                      class="item"
                      id="1"
                    >
                      news
                    </div>
                  </a>
                  <a
                    href="/content/standings"
                  >
                    <div
                      class="item"
                      id="2"
                    >
                      standings
                    </div>
                  </a>
                  <a
                    href="/content/stats"
                  >
                    <div
                      class="item"
                      id="3"
                    >
                      stats
                    </div>
                  </a>
                  <a
                    href="/content/players"
                  >
                    <div
                      class="item"
                      id="4"
                    >
                      players
                    </div>
                  </a>
                </div>
              </div>
            </div>
            <a
              href="/"
            >
              <span
                role="button"
              >
                <svg
                  class="bi bi-x"
                  fill="white"
                  height="2em"
                  viewBox="0 0 16 16"
                  width="2em"
                  xmlns="http://www.w3.org/2000/svg"
                >
                  <path
                    d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"
                    fill-rule="evenodd"
                  />
                </svg>
              </span>
            </a>
          </div>
          <div
            class="row no-gutters"
          >
            <div
              class="col-md-12"
            >
              <div
                class="row no-gutters"
              >
                <div
                  class="col-md-6 border"
                >
                  <div
                    class="row no-gutters row-item"
                  >
                    <div
                      class="col-8 col-md-8 p-2"
                    >
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="crystal-palace.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Crystal Palace
                        </div>
                        <div
                          class="col-1 col-md-1"
                        >
                          0
                        </div>
                      </div>
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="newcastle.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Newcastle
                        </div>
                        <div
                          class="col-1 col-md-1"
                        >
                          2
                        </div>
                      </div>
                    </div>
                    <div
                      class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                    >
                      <div>
                        FT
                      </div>
                      <div>
                        Yesterday
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="col-md-6 border"
                >
                  <div
                    class="row no-gutters row-item"
                  >
                    <div
                      class="col-8 col-md-8 p-2"
                    >
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="brighton.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Brighton
                        </div>
                        <div
                          class="col-1 col-md-1"
                        >
                          1
                        </div>
                      </div>
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="liverpool.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Liverpool
                        </div>
                        <div
                          class="col-1 col-md-1"
                        >
                          3
                        </div>
                      </div>
                    </div>
                    <div
                      class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                    >
                      <div>
                        FT
                      </div>
                      <div>
                        Yesterday
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="row no-gutters"
              >
                <div
                  class="col-md-6 border"
                >
                  <div
                    class="row no-gutters row-item"
                  >
                    <div
                      class="col-8 col-md-8 p-2"
                    >
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="man-city.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Man. City
                        </div>
                        <div
                          class="col-1 col-md-1"
                        >
                          2
                        </div>
                      </div>
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="burnley.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Burnley
                        </div>
                        <div
                          class="col-1 col-md-1"
                        >
                          1
                        </div>
                      </div>
                    </div>
                    <div
                      class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                    >
                      <div>
                        FT
                      </div>
                      <div>
                        Yesterday
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="col-md-6 border"
                >
                  <div
                    class="row no-gutters row-item"
                  >
                    <div
                      class="col-8 col-md-8 p-2"
                    >
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="everton.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Everton
                        </div>
                        <div
                          class="col-1 col-md-1"
                        />
                      </div>
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="leeds.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Leeds United
                        </div>
                        <div
                          class="col-1 col-md-1"
                        />
                      </div>
                    </div>
                    <div
                      class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                    >
                      <div>
                        Today
                      </div>
                      <div>
                        19:30
                      </div>
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="row no-gutters"
              >
                <div
                  class="col-md-6 border"
                >
                  <div
                    class="row no-gutters row-item"
                  >
                    <div
                      class="col-8 col-md-8 p-2"
                    >
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="west-brom.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          West Brom
                        </div>
                        <div
                          class="col-1 col-md-1"
                        />
                      </div>
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="sheffield.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Sheffield United
                        </div>
                        <div
                          class="col-1 col-md-1"
                        />
                      </div>
                    </div>
                    <div
                      class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                    >
                      <div>
                        Today
                      </div>
                      <div>
                        22:00
                      </div>
                    </div>
                  </div>
                </div>
                <div
                  class="col-md-6 border"
                >
                  <div
                    class="row no-gutters row-item"
                  >
                    <div
                      class="col-8 col-md-8 p-2"
                    >
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="southampton.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Southampton
                        </div>
                        <div
                          class="col-1 col-md-1"
                        />
                      </div>
                      <div
                        class="row justify-content-center m-1"
                      >
                        <div
                          class="col-2 col-md-2"
                        >
                          <img
                            src="man-utd.png"
                            width="25"
                          />
                        </div>
                        <div
                          class="col-8 col-md-8"
                        >
                          Man United
                        </div>
                        <div
                          class="col-1 col-md-1"
                        />
                      </div>
                    </div>
                    <div
                      class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                    >
                      <div>
                        Tomorrow
                      </div>
                      <div>
                        16:00
                      </div>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="row no-gutters"
    >
      <div
        class="col-md-12"
      >
        <div
          class="row flex-nowrap overflow-auto no-gutters content-header"
        >
          <div
            class="d-flex"
          >
            <a
              href="/"
            >
              <svg
                class="bi bi-arrow-left-short"
                fill="white"
                height="2em"
                viewBox="0 0 16 16"
                width="2em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M12 8a.5.5 0 0 1-.5.5H5.707l2.147 2.146a.5.5 0 0 1-.708.708l-3-3a.5.5 0 0 1 0-.708l3-3a.5.5 0 1 1 .708.708L5.707 7.5H11.5a.5.5 0 0 1 .5.5z"
                  fill-rule="evenodd"
                />
              </svg>
            </a>
            <div
              class="m-1"
            >
              Premier League
            </div>
          </div>
          <div
            class="row flex-nowrap overflow-auto"
          >
            <div
              class="col-md-12 mt-2"
            >
              <div
                class="row flex-nowrap overflow-auto ml-1"
              >
                <a
                  href="/content/matches"
                >
                  <div
                    class="border-bottom border-white item"
                    id="0"
                  >
                    matches
                  </div>
                </a>
                <a
                  href="/content/news"
                >
                  <div
                    class="item"
                    id="1"
                  >
                    news
                  </div>
                </a>
                <a
                  href="/content/standings"
                >
                  <div
                    class="item"
                    id="2"
                  >
                    standings
                  </div>
                </a>
                <a
                  href="/content/stats"
                >
                  <div
                    class="item"
                    id="3"
                  >
                    stats
                  </div>
                </a>
                <a
                  href="/content/players"
                >
                  <div
                    class="item"
                    id="4"
                  >
                    players
                  </div>
                </a>
              </div>
            </div>
          </div>
          <a
            href="/"
          >
            <span
              role="button"
            >
              <svg
                class="bi bi-x"
                fill="white"
                height="2em"
                viewBox="0 0 16 16"
                width="2em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M4.646 4.646a.5.5 0 0 1 .708 0L8 7.293l2.646-2.647a.5.5 0 0 1 .708.708L8.707 8l2.647 2.646a.5.5 0 0 1-.708.708L8 8.707l-2.646 2.647a.5.5 0 0 1-.708-.708L7.293 8 4.646 5.354a.5.5 0 0 1 0-.708z"
                  fill-rule="evenodd"
                />
              </svg>
            </span>
          </a>
        </div>
        <div
          class="row no-gutters"
        >
          <div
            class="col-md-12"
          >
            <div
              class="row no-gutters"
            >
              <div
                class="col-md-6 border"
              >
                <div
                  class="row no-gutters row-item"
                >
                  <div
                    class="col-8 col-md-8 p-2"
                  >
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="crystal-palace.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Crystal Palace
                      </div>
                      <div
                        class="col-1 col-md-1"
                      >
                        0
                      </div>
                    </div>
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="newcastle.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Newcastle
                      </div>
                      <div
                        class="col-1 col-md-1"
                      >
                        2
                      </div>
                    </div>
                  </div>
                  <div
                    class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                  >
                    <div>
                      FT
                    </div>
                    <div>
                      Yesterday
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="col-md-6 border"
              >
                <div
                  class="row no-gutters row-item"
                >
                  <div
                    class="col-8 col-md-8 p-2"
                  >
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="brighton.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Brighton
                      </div>
                      <div
                        class="col-1 col-md-1"
                      >
                        1
                      </div>
                    </div>
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="liverpool.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Liverpool
                      </div>
                      <div
                        class="col-1 col-md-1"
                      >
                        3
                      </div>
                    </div>
                  </div>
                  <div
                    class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                  >
                    <div>
                      FT
                    </div>
                    <div>
                      Yesterday
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="row no-gutters"
            >
              <div
                class="col-md-6 border"
              >
                <div
                  class="row no-gutters row-item"
                >
                  <div
                    class="col-8 col-md-8 p-2"
                  >
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="man-city.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Man. City
                      </div>
                      <div
                        class="col-1 col-md-1"
                      >
                        2
                      </div>
                    </div>
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="burnley.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Burnley
                      </div>
                      <div
                        class="col-1 col-md-1"
                      >
                        1
                      </div>
                    </div>
                  </div>
                  <div
                    class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                  >
                    <div>
                      FT
                    </div>
                    <div>
                      Yesterday
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="col-md-6 border"
              >
                <div
                  class="row no-gutters row-item"
                >
                  <div
                    class="col-8 col-md-8 p-2"
                  >
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="everton.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Everton
                      </div>
                      <div
                        class="col-1 col-md-1"
                      />
                    </div>
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="leeds.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Leeds United
                      </div>
                      <div
                        class="col-1 col-md-1"
                      />
                    </div>
                  </div>
                  <div
                    class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                  >
                    <div>
                      Today
                    </div>
                    <div>
                      19:30
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="row no-gutters"
            >
              <div
                class="col-md-6 border"
              >
                <div
                  class="row no-gutters row-item"
                >
                  <div
                    class="col-8 col-md-8 p-2"
                  >
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="west-brom.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        West Brom
                      </div>
                      <div
                        class="col-1 col-md-1"
                      />
                    </div>
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="sheffield.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Sheffield United
                      </div>
                      <div
                        class="col-1 col-md-1"
                      />
                    </div>
                  </div>
                  <div
                    class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                  >
                    <div>
                      Today
                    </div>
                    <div>
                      22:00
                    </div>
                  </div>
                </div>
              </div>
              <div
                class="col-md-6 border"
              >
                <div
                  class="row no-gutters row-item"
                >
                  <div
                    class="col-8 col-md-8 p-2"
                  >
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="southampton.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Southampton
                      </div>
                      <div
                        class="col-1 col-md-1"
                      />
                    </div>
                    <div
                      class="row justify-content-center m-1"
                    >
                      <div
                        class="col-2 col-md-2"
                      >
                        <img
                          src="man-utd.png"
                          width="25"
                        />
                      </div>
                      <div
                        class="col-8 col-md-8"
                      >
                        Man United
                      </div>
                      <div
                        class="col-1 col-md-1"
                      />
                    </div>
                  </div>
                  <div
                    class="col-4 col-md-4 d-flex flex-column justify-content-center text-center"
                  >
                    <div>
                      Tomorrow
                    </div>
                    <div>
                      16:00
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
